.azure-pipelines.yml

```diff
@@ -94,6 +94,10 @@ jobs:
         if git log origin/master --exit-code -- "$FILENAME" > /dev/null
         then exit 0; fi
       done
+      # Special case if we are the first commit after a tag (commit that
+      # set `+dev` version prefix, hence no newsfragment provided)
+      if ([[ $(git log --oneline $(git describe --tags --abbrev=0).. | wc -l) -eq 1 ]]);
+      then exit 0; fi
```

Comment:
Looks a bit fragile to me, maybe simply run the newsfragment check if it comes from a PR (since the `+dev` commit can be pushed directly to master).

Reply:
👍 see 5127369bc24f0e085cb0c56682cf0186993cbda6